### PR TITLE
Update tfm-rubygem-jwt to 2.2.1

### PR DESCRIPTION
**Pull request by rahulbajaj0509**

### CODE REVIEW

File: packages/foreman/tfm-rubygem-jwt/tfm-rubygem-jwt.spec [hunk 1]
```diff
@@ -69,14 +69,10 @@ cp -pa .%{gem_dir}/* \
 
 %files
 %dir %{gem_instdir}
-%{gem_instdir}/.codeclimate.yml
```
Comment (ekohl) [inline packages/foreman/tfm-rubygem-jwt/tfm-rubygem-jwt.spec]: I suggested to add `%exclude` to it. These files are still part of the gem but we don't want to ship these in the RPM. Same for `.ebert.yml`.

File: packages/foreman/tfm-rubygem-jwt/tfm-rubygem-jwt.spec [hunk 2]
```diff
@@ -92,6 +88,9 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/spec
```
Comment (ekohl) [inline packages/foreman/tfm-rubygem-jwt/tfm-rubygem-jwt.spec]: This line should also go since the gem no longer ships this.

**Comment by rahulbajaj0509** [comment]
@ekohl 
```
Checking for unpackaged file(s): /usr/lib/rpm/check-files /builddir/build/BUILDROOT/tfm-rubygem-jwt-2.2.1-1.el7.noarch
BUILDSTDERR: error: Installed (but unpackaged) file(s) found:
BUILDSTDERR:    /opt/theforeman/tfm/root/usr/share/gems/gems/jwt-2.2.1/AUTHORS
BUILDSTDERR:    /opt/theforeman/tfm/root/usr/share/gems/gems/jwt-2.2.1/Appraisals
BUILDSTDERR:     Installed (but unpackaged) file(s) found:
BUILDSTDERR:    /opt/theforeman/tfm/root/usr/share/gems/gems/jwt-2.2.1/AUTHORS
BUILDSTDERR:    /opt/theforeman/tfm/root/usr/share/gems/gems/jwt-2.2.1/Appraisals
RPM build errors:
Child return code was: 1
EXCEPTION: [Error()]
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/mockbuild/trace_decorator.py", line 96, in trace
    result = func(*args, **kw)
  File "/usr/lib/python2.7/site-packages/mockbuild/util.py", line 634, in do
    raise exception.Error("Command failed: \n # %s\n%s" % (command, output), child.returncode)
Error: Command failed: 
 # bash --login -c /usr/bin/rpmbuild -bb --target noarch --nodeps /builddir/build/SPECS/tfm-rubygem-jwt.spec
```

How do go about understanding this?

**Comment by ekohl** [comment]
Those are the inverse: they are files that are packaged but not mentioned in `%files`. In this case I'd add it under `%files doc` so they're part of the `-doc` subpackage. I'd also prefix them with `%doc` so RPM correctly recognizes it as doc files.

**Comment by rahulbajaj0509** [comment]
@ekohl that worked out, atleast koji test passed.
```
[2019-10-09T16:59:46.846Z]     package: foreman-1.24.0-0.7.develop.20191008105239git0b3ca6c.el7.noarch from el7-foreman-nightly

[2019-10-09T16:59:46.846Z]       unresolved deps:

[2019-10-09T16:59:46.846Z]          tfm-rubygem(jwt) < 0:2.2.0

[2019-10-09T16:59:46.846Z]   stdout_lines: <omitted>
```
This is the error I am facing now. Any heads-up on how to resolve this one?

**Comment by ekohl** [comment]
Related core PR: https://github.com/theforeman/foreman/pull/6549

**Pull request merged**